### ISSUE TITLE
[1.18] Easy Villagers/piglins mods : Villagers and piglins can be picked up in claims 

### ISSUE THREAD

**Issue by jinkhya** [thexaero/open-parties-and-claims#232]
Hello,
Easy villager and Easy Piglins mods bypass the claim system : you can shift right click them to pick them up. I can't right click or left click but I can shift right click to steal them. I dunno if you can do anything about this, or if I can prevent this somehow in the config. 

Open Parties and Claims 0.17.2
EasyVillagers : 1.0.11
EasyPiglins : 1.0.4

Thanks in advance,
Jin

**Comment by thexaero** [comment]
Yeah, something like this would need custom protection. They're not firing the entity interaction event because it's not a normal interaction.
